src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -419,32 +411,28 @@ async def initialize(resampler_config: ResamplerConfig) -> None:
 
 
 def frequency() -> GridFrequency:
-    """Return the grid frequency.
-
-    Returns:
-        The grid frequency.
-    """
-    return _get().frequency()
+    """Return the grid frequency measuring point."""
+    return _get().frequency
 
 
 def voltage() -> VoltageStreamer:
-    """Return the 3-phase voltage for the microgrid.
-
-    Returns:
-        The 3-phase voltage.
-    """
-    return _get().voltage()
+    """Return the 3-phase voltage measuring point."""
+    return _get().voltage
 
 
 def logical_meter() -> LogicalMeter:
-    """Return the logical meter instance.
+    """Return the logical meter of the microgrid."""
+    return _get().logical_meter
 
-    If a LogicalMeter instance doesn't exist, a new one is created and returned.
 
-    Returns:
-        A logical meter instance.
-    """
-    return _get().logical_meter()
+def consumer() -> Consumer:
+    """Return the [`Consumption`][frequenz.sdk.timeseries.consumer.Consumer] measuring point."""
+    return _get().consumer
+
+
+def producer() -> Producer:
+    """Return the [`Production`][frequenz.sdk.timeseries.producer.Producer] measuring point."""
+    return _get().producer
```

Comment:
Maybe I am lacking context from the diff, but shouldn't all these be also `@property`?

Reply:
Oh, so the lazy ones are internal, and this is the public interface, right? So the other changes are not visible to the user and these will be. If so I think we should do it but in a different PR.

Reply:
these are functions in the `microgrid` module.  They are not class methods, so they can't be properties.

Reply:
Right, yeah, I find that weird, also at some point I think we should make `microgrid` a proper object (with a flobal instance) instead of having global functions, but that's for another time...